src/styles/form.css

```diff
@@ -0,0 +1,120 @@
+.embeddable-content div.form_question {
```

Comment:
You know sass strategies work here :wink:

Reply:
It does?! 😲 WOOHOO! I'll get this fixup